grafana/grafana
# Logging: Adds x-cache response header when present

fixes https://github.com/grafana/grafana-enterprise/issues/4616

**Changes**
Adds the `X-Cache` response header to the context log when present.

Reasons for change described on the issue.

**How to test**
- make sure query caching is enabled
- cause some panel queries that get served by the cache
- look at your context logs and you should see either an `x-cache=HIT` or `x-cache=MISS` label
